docker-compose.yml.local

```diff
@@ -2,14 +2,41 @@ version: '3'
 
 services:
 
-  php:
-    image: yiisoftware/yii2-php:7.2-apache
```

Comment:
Can't we use 7.4-apache?

Reply:
yiisoftware/yii2-php:7.4-apache not found.
error: manifest for yiisoftware/yii2-php:7.4-apache not found: manifest unknown: manifest unknown

Reply:
@rezaei121 You don't need `yii2` images. See https://hub.docker.com/r/yiisoftware/yii-php/tags

Reply:
thank you @xepozz .
@samdark I updated pull request. new commit.